<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompLinePack</class>
 <widget class="QWidget" name="CompLinePack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>76</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,0,3">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLineEdit" name="txtSendText">
       <property name="maxLength">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chbxTimer">
       <property name="text">
        <string>Timer (ms)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="numudInterval">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="rdbDecimal">
       <property name="text">
        <string>Decimal</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rdbHex">
       <property name="text">
        <string>Hex</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rdbASCII">
       <property name="text">
        <string>ASCII</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chbxFile">
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFile">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;File Template ( All in a single Line ) :&lt;/span&gt;&lt;/p&gt;&lt;p&gt;if Hex is Checked :&lt;/p&gt;&lt;p&gt;12 2A 4b Fc ...&lt;/p&gt;&lt;p&gt;if Decimal is Checked :&lt;/p&gt;&lt;p&gt;100 213 0 32 ...&lt;/p&gt;&lt;p&gt;if ASCII is Checked :&lt;/p&gt;&lt;p&gt;ASDDF VC&amp;lt;XC JFISLA ...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblFileName">
       <property name="text">
        <string>----------</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSend">
       <property name="text">
        <string>Send</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
